### PR TITLE
Improve tests

### PR DESCRIPTION
**Pull request by Grubas7**
Added tests for `-ss_defaultFontName`.
Improved tests for `-ss_defaultBaseSize` and `-preferredFontSizeDelta`.

### CODE REVIEW

**Comment by jhersh** [comment]
Neat! We could probably increase coverage even further by deallocating the views being tested, perhaps in `-tearDown`.

**Comment by Grubas7** [comment]
Yeap, I'll write more test over next few days.

File: Example/SSDynamicTextTests/SSDynamicViewsReleaseTests.m [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    @autoreleasepool {
+        SSDynamicLabel *label = [SSDynamicLabel labelWithFont:SSTestFontName baseSize:SSTestFontSize];
+        [label release];
+    }
+
+    // Act & Assert
+    XCTAssertNoThrow([SSTestsHelper postContentSizeChangeNotification]);
+}
+
+//- (void)testDynamicButtonShouldNotObserveTextSizeChangesAfterRelease {
```
Comment (Grubas7) [inline Example/SSDynamicTextTests/SSDynamicViewsReleaseTests.m]: I don't have idea why tests for button hangs.

**Comment by jhersh** [comment]
Thanks! Feel free to squash and merge. :+1:

**Comment by Grubas7** [comment]
Squashed.

**Comment by Grubas7** [comment]
Squashed.

I checked one more time SSDynamicButton release tests on CI. I don't know why this problem occur. I'll check this and make new PR with fix.

**Pull request merged**